views/components/Certification.js

```diff
@@ -1,72 +1,418 @@
-          </li>
-          <li className="process__step-info process__step-info_develop">
-            <h3>
-              <spana>2</spana>
-              &mdash;Communicative Certification: TOEFL ibt 85 pts
-            </h3>
-            <p>
-              The TOEFL iBT® test measures your ability to use and understand
-              English at the university level. And it evaluates how well you
-              combine your reading, listening, speaking and writing skills to
-              perform academic tasks.
-            </p>
-          </li>
-          <li className="process__step process__step_deploy">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deploy">
-            <h3>
-              <spana>3</spana>
-              &mdash;Second place
-            </h3>
-            <p>
-              Got second prize for the competitive algorithm contest of the
-              university
-            </p>
-          </li>
-          <li className="process__step process__step_deliver">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deliver">
-            <h3>
-              <spana>4</spana>
-              &mdash;First place
-            </h3>
-            <p>
-              Highschool, got First prize for the competitive algorithm contest
-              of the city.
-            </p>
-          </li>
-        </ul>
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+import Icon from '~/views/components/common/Icon'
+import { CERTIFICATIONS } from '~/consts/pages'
+import { INIT_CERT_ACTIVE_ITEM } from '~/consts/initState'
+import PropTypes from 'prop-types'
+
+const Certification = () => {
+  const [itemSelected, setSelectedItem] = useState(INIT_CERT_ACTIVE_ITEM[0])
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <header className="common-section__header">
+          <h2>Certifications and Prizes</h2>
+          <h4></h4>
+        </header>
+        <div className="grid-container">
+          <List className="certification">
+            <Research
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Develop
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deploy
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deliver
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+          </List>
+        </div>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
+
+const Research = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <>
```

Comment:
It's ok but preferable using:
`import React, {Fragment} from ...`
`<Fragment> `
...
`</Fragment> `

Reply:
Updated